indico/modules/events/registration/forms.py

```diff
@@ -278,12 +279,31 @@ def validate_users_field(self, field):
                                   .format(emails=', '.join(sorted(existing))))
 
 
+class CSVFieldDelimiter(RichIntEnum):
```

Comment:
Not sure if I ever tested it, but see if `RichStrEnum` works here as well (I think it does). That would allow using nicer enum member values:

```
    comma = auto()
    semicolon = auto()
    tab = auto()
    space = auto()
```

Which gives you the enum member name as the value.
Of course `__titles__` then needs to be a dict, but that shouldn't be a problem ;)

Reply:
Had to change a few things, but it works.